pynbody/plot/sph.py

```diff
@@ -111,11 +111,11 @@ def velocity_image(sim, width="10 kpc", vector_color='black', edgecolor='black',
             width = _units.Unit(width)
         width = width.in_units(sim['pos'].units, **sim.conversion_context())
 
-    width = float(width)
+    width = np.float32(width)
```

Comment:
Why should this specifically be `np.float32` rather than just `float`? I don't quite get it. (Similar point on the line below)